sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/Program.cs

```diff
@@ -93,12 +93,8 @@ private static DigitalTwinsClient GetDigitalTwinsClient(string tenantId, string
 
             // By using the ClientSecretCredential, a specified application Id can login using a
             // client secret.
-            var tokenCredential = new ClientSecretCredential(
-                tenantId,
-                clientId,
-                clientSecret,
-                new TokenCredentialOptions { AuthorityHost = KnownAuthorityHosts.AzureCloud });
-
+            var tokenCredential = new DefaultAzureCredential();
```

Comment:
This requires the following environment variables to be set. I chose to set it in Visual Studio debug settings as the Azure.Core team told us not to expose too many details about Azure.Identity to the user. I personally think it is hard for user to understand and might be slightly better to do Environment.Set in the snippet. What do you think @drwill-ms ?
AZURE_CLIENT_ID
AZURE_TENANT_ID
AZURE_CLIENT_SECRET

Reply:
Agreed we should use Environment.Set, unless those values are already set. That doesn't have to be in the snippet.